src/lib/Search/Legacy/Content/Common/Gateway/CriterionHandler/ContentName.php

```diff
@@ -0,0 +1,144 @@
+<?php
+
+/**
+ * @copyright Copyright (C) Ibexa AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace Ibexa\Core\Search\Legacy\Content\Common\Gateway\CriterionHandler;
+
+use Doctrine\DBAL\Connection;
+use Doctrine\DBAL\Query\QueryBuilder;
+use Ibexa\Contracts\Core\Repository\Values\Content\Query\Criterion;
+use Ibexa\Core\Persistence\TransformationProcessor;
+use Ibexa\Core\Search\Legacy\Content\Common\Gateway\CriteriaConverter;
+use Ibexa\Core\Search\Legacy\Content\Common\Gateway\CriterionHandler;
+
+final class ContentName extends CriterionHandler
+{
+    private const EZCONTENTOBJECT_NAME_ALIAS = 'ezc_n';
+    private const EZCONTENTOBJECT_ALIAS = 'c';
```

Comment:
I would personally avoid any legacy references even if they are accurate and correspond with the old database naming:
```suggestion
    private const CONTENTOBJECT_NAME_ALIAS = 'ezc_n';
    private const CONTENTOBJECT_ALIAS = 'c';
```

Reply:
https://github.com/ibexa/core/pull/312/commits/94bf067423b5b9928c0d350d72c2f87bce286bc4